### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by IntelOrca**
VS2019 is now required.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -84,13 +84,8 @@ Some \*BSD operating systems offer native packages. These packages are usually t
 OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bought at either [Steam](http://store.steampowered.com/app/285330/) or [GOG.com](http://www.gog.com/game/rollercoaster_tycoon_2).
 
 ### Windows:
-- 7 / 8 / 10
-- Visual Studio 2017 update 7 (Enterprise / Professional / [Community (Free)](https://www.visualstudio.com/vs/community/))
+- Visual Studio 2019 (Enterprise / Professional / [Community (Free)](https://www.visualstudio.com/vs/community/))
   - Desktop development with C++
-  - [Windows 10 SDK (10.0.14393.0)](https://go.microsoft.com/fwlink/p/?LinkId=838916)
-- [7-Zip](http://www.7-zip.org/) (for deployment only)
-- [NSIS](http://nsis.sourceforge.net/) (for deployment only)
```
Comment (Gymnasiast) [inline readme.md]: Why were these lines removed?

Comment (IntelOrca) [reply readme.md]: The Windows 10 SDK is installed by default with C++ workload.
7-zip and NSIS are not necessary, they are only used by CI - so simpler to not confuse new developers.

Just keep it simple. Install VS 2019, checkout, build. DONE.

**Comment by James103** [comment]
One of the checks (android - push) has failed, it threw a `Exception in thread "main" javax.net.ssl.SSLException: Connection has been shutdown: javax.net.ssl.SSLException: java.net.SocketException: Connection reset`. Why?

**Comment by IntelOrca** [comment]
@James103 I don't know but any job that downloads a third party file is susceptible to failing from time to time. We will have to monitor how often this happens. It however has nothing to do with this change.
